backend/src/main/java/Matchmaking/Team.java

```diff
@@ -0,0 +1,45 @@
+package Matchmaking;
+
+import Matchmaking.Player;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Team {
+  private List<Player> players;
+  private float avgSkill;
+  private int size;
+
+  public Team() {
+    this.players = new ArrayList<>();
+    this.avgSkill = 0;
+    this.size = 0;
```

Comment:
Out of curiosity, why did you add and update a size variable when this is reflected in the arraylist?